
We&#39;re here today to announce
the first synthetic cell,
a cell made by
starting with the digital code in the computer,
building the chromosome
from four bottles of chemicals,
assembling that chromosome in yeast,
transplanting it into
a recipient bacterial cell
and transforming that cell
into a new bacterial species.
So this is the first self-replicating species
that we&#39;ve had on the planet
whose parent is a computer.
It also is the first species
to have its own website
encoded in its genetic code.
But we&#39;ll talk more about
the watermarks in a minute.
This is a project that had its inception
15 years ago
when our team then --
we called the institute TIGR --
was involved in sequencing
the first two genomes in history.
We did Haemophilus influenzae
and then the smallest genome of a self-replicating organism,
that of Mycoplasma genitalium.
And as soon as
we had these two sequences
we thought, if this is supposed to be the smallest genome
of a self-replicating species,
could there be even a smaller genome?
Could we understand the basis of cellular life
at the genetic level?
It&#39;s been a 15-year quest
just to get to the starting point now
to be able to answer those questions,
because it&#39;s very difficult to eliminate
multiple genes from a cell.
You can only do them one at a time.
We decided early on
that we had to take a synthetic route,
even though nobody had been there before,
to see if we could synthesize
a bacterial chromosome
so we could actually vary the gene content
to understand the essential genes for life.
That started our 15-year quest
to get here.
But before we did the first experiments,
we actually asked
Art Caplan&#39;s team at the University of Pennsylvania
to undertake a review
of what the risks, the challenges,
the ethics around creating new
species in the laboratory were
because it hadn&#39;t been done before.
They spent about two years
reviewing that independently
and published their results in Science in 1999.
Ham and I took two years off
as a side project to sequence the human genome,
but as soon as that was done
we got back to the task at hand.
In 2002, we started
a new institute,
the Institute for Biological Energy Alternatives,

where we set out two goals:
One, to understand
the impact of our technology on the environment,
and how to understand the environment better,
and two, to start down this process
of making synthetic life
to understand basic life.
In 2003,
we published our first success.
So Ham Smith and Clyde Hutchison
developed some new methods
for making error-free DNA
at a small level.
Our first task was
a 5,000-letter code bacteriophage,
a virus that attacks only E. coli.
So that was
the phage phi X 174,
which was chosen for historical reasons.
It was the first DNA phage,
DNA virus, DNA genome
that was actually sequenced.
So once we realized
that we could make 5,000-base pair
viral-sized pieces,
we thought, we at least have the means
then to try and make serially lots of these pieces
to be able to eventually assemble them together
to make this mega base chromosome.
So, substantially larger than
we even thought we would go initially.

There were several steps to this. There were two sides:
We had to solve the chemistry
for making large DNA molecules,
and we had to solve the biological side
of how, if we had this new chemical entity,
how would we boot it up, activate it
in a recipient cell.

We had two teams working in parallel:
one team on the chemistry,
and the other on trying to
be able to transplant
entire chromosomes
to get new cells.
When we started this out, we thought the synthesis would be the biggest problem,
which is why we chose the smallest genome.
And some of you have noticed that we switched from the smallest genome
to a much larger one.
And we can walk through the reasons for that,
but basically the small cell
took on the order of
one to two months to get results from,
whereas the larger, faster-growing cell
takes only two days.
So there&#39;s only so many cycles we could go through
in a year at six weeks per cycle.
And you should know that basically
99, probably 99 percent plus
of our experiments failed.
So this was a debugging,
problem-solving scenario from the beginning
because there was no recipe
of how to get there.
So, one of the most important publications we had
was in 2007.
Carole Lartigue led the effort
to actually transplant a bacterial chromosome
from one bacteria to another.
I think philosophically, that was one of the most important papers
that we&#39;ve ever done
because it showed how dynamic life was.
And we knew, once that worked,
that we actually had a chance
if we could make the synthetic chromosomes
to do the same with those.
We didn&#39;t know that it was going to take us
several years more to get there.
In 2008,
we reported the complete synthesis
of the Mycoplasma genitalium genome,
a little over 500,000 letters of genetic code,
but we have not yet succeeded in booting up that chromosome.
We think in part, because of its slow growth
and, in part,
cells have all kinds of unique defense mechanisms
to keep these events from happening.
It turned out the cell that we were trying to transplant into
had a nuclease, an enzyme that chews up DNA on its surface,
and was happy to eat
the synthetic DNA that we gave it
and never got transplantations.
But at the time, that was the largest
molecule of a defined structure
that had been made.
And so both sides were progressing,
but part of the synthesis
had to be accomplished or was able to be accomplished
using yeast, putting the fragments in yeast
and yeast would assemble these for us.
It&#39;s an amazing step forward,
but we had a problem because now we had
the bacterial chromosomes growing in yeast.
So in addition to doing the transplant,
we had to find out how to get a bacterial chromosome
out of the eukaryotic yeast
into a form where we could transplant it
into a recipient cell.
So our team developed new techniques
for actually growing, cloning
entire bacterial chromosomes in yeast.
So we took the same mycoides genome
that Carole had initially transplanted,
and we grew that in yeast
as an artificial chromosome.
And we thought this would be a great test bed
for learning how to get chromosomes out of yeast
and transplant them.
When we did these experiments, though,
we could get the chromosome out of yeast
but it wouldn&#39;t transplant and boot up a cell.
That little issue took the team two years to solve.
It turns out, the DNA in the bacterial cell
was actually methylated,
and the methylation protects it from the restriction enzyme,
from digesting the DNA.
So what we found is if we took the chromosome
out of yeast and methylated it,
we could then transplant it.
Further advances came
when the team removed the restriction enzyme genes
from the recipient capricolum cell.
And once we had done that, now we can take
naked DNA out of yeast and transplant it.
So last fall
when we published the results of that work in Science,
we all became overconfident
and were sure we were only
a few weeks away
from being able to now boot up
a chromosome out of yeast.
Because of the problems with
Mycoplasma genitalium and its slow growth
about a year and a half ago,
we decided to synthesize
the much larger chromosome, the mycoides chromosome,
knowing that we had the biology worked out on that
for transplantation.
And Dan led the team for the synthesis
of this over one-million-base pair chromosome.
But it turned out it wasn&#39;t going to be as simple in the end,
and it set us back three months
because we had one error
out of over a million base pairs in that sequence.
So the team developed new debugging software,
where we could test each synthetic fragment
to see if it would grow in a background
of wild type DNA.
And we found that 10 out of the 11
100,000-base pair pieces we synthesized
were completely accurate
and compatible with
a life-forming sequence.
We narrowed it down to one fragment;
we sequenced it
and found just one base pair had been deleted
in an essential gene.
So accuracy is essential.
There&#39;s parts of the genome
where it cannot tolerate even a single error,
and then there&#39;s parts of the genome
where we can put in large blocks of DNA,
as we did with the watermarks,
and it can tolerate all kinds of errors.
So it took about three months to find that error
and repair it.
And then early one morning, at 6 a.m.
we got a text from Dan
saying that, now, the first blue colonies existed.

So, it&#39;s been a long route to get here:
15 years from the beginning.
We felt
one of the tenets of this field
was to make absolutely certain
we could distinguish synthetic DNA
from natural DNA.
Early on, when you&#39;re working in a new area of science,
you have to think about all the pitfalls
and things that could lead you
to believe that you had done something when you hadn&#39;t,
and, even worse, leading others to believe it.
So, we thought the worst problem would be
a single molecule contamination
of the native chromosome,
leading us to believe that we actually had
created a synthetic cell,
when it would have been just a contaminant.
So early on, we developed the notion
of putting in watermarks in the DNA
to absolutely make clear
that the DNA was synthetic.
And the first chromosome we built
in 2008 --
the 500,000-base pair one --
we simply assigned
the names of the authors of the chromosome
into the genetic code,
but it was using just amino acid
single letter translations,
which leaves out certain letters of the alphabet.
So the team actually developed a new code
within the code within the code.
So it&#39;s a new code
for interpreting and writing messages in DNA.
Now, mathematicians have been hiding and writing
messages in the genetic code for a long time,
but it&#39;s clear they were mathematicians and not biologists
because, if you write long messages
with the code that the mathematicians developed,
it would more than likely lead to
new proteins being synthesized
with unknown functions.
So the code that Mike Montague and the team developed
actually puts frequent stop codons,
so it&#39;s a different alphabet
but allows us to use
the entire English alphabet
with punctuation and numbers.
So, there are four major watermarks
all over 1,000 base pairs of genetic code.
The first one actually contains within it
this code for interpreting
the rest of the genetic code.
So in the remaining information,
in the watermarks,
contain the names of, I think it&#39;s
46 different authors
and key contributors
to getting the project to this stage.
And we also built in
a website address
so that if somebody decodes the code
within the code within the code,
they can send an email to that address.
So it&#39;s clearly distinguishable
from any other species,
having 46 names in it,
its own web address.
And we added three quotations,
because with the first genome
we were criticized for not trying to say something more profound
than just signing the work.
So we won&#39;t give the rest of the code,
but we will give the three quotations.
The first is,
&quot;To live, to err,
to fall, to triumph
and to recreate life out of life.&quot;
It&#39;s a James Joyce quote.
The second quotation is, &quot;See things not as they are,
but as they might be.&quot;
It&#39;s a quote from the &quot;American Prometheus&quot;
book on Robert Oppenheimer.

And the last one is a Richard Feynman quote:
&quot;What I cannot build,
I cannot understand.&quot;
So, because this is as much a philosophical advance
as a technical advance in science,
we tried to deal with both the philosophical
and the technical side.
The last thing I want to say before turning it over to questions
is that the extensive work
that we&#39;ve done --
asking for ethical review,
pushing the envelope
on that side as well as the technical side --
this has been broadly discussed in the scientific community,
in the policy community
and at the highest levels of the federal government.
Even with this announcement,
as we did in 2003 --
that work was funded by the Department of Energy,
so the work was reviewed
at the level of the White House,
trying to decide whether to classify the work or publish it.
And they came down on the side of open publication,
which is the right approach --
we&#39;ve briefed the White House,
we&#39;ve briefed members of Congress,
we&#39;ve tried to take and push
the policy issues
in parallel with the scientific advances.
So with that, I would like
to open it first to the floor for questions.
Yes, in the back.

Reporter: Could you explain, in layman&#39;s terms,
how significant a breakthrough this is please?

Craig Venter: Can we explain how significant this is?
I&#39;m not sure we&#39;re the ones that should be explaining how significant it is.
It&#39;s significant to us.
Perhaps it&#39;s a giant philosophical change
in how we view life.
We actually view it as a baby step in terms of,
it&#39;s taken us 15 years to be able
to do the experiment
we wanted to do 15 years ago
on understanding life at its basic level.
But we actually believe
this is going to be a very powerful set of tools
and we&#39;re already starting
in numerous avenues
to use this tool.
We have, at the Institute,
ongoing funding now from NIH
in a program with Novartis
to try and use these new
synthetic DNA tools
to perhaps make the flu vaccine
that you might get next year.
Because instead of taking weeks to months to make these,
Dan&#39;s team can now make these
in less than 24 hours.
So when you see how long it took to get an H1N1 vaccine out,
we think we can shorten that process
quite substantially.
In the vaccine area,
Synthetic Genomics and the Institute
are forming a new vaccine company
because we think these tools can affect vaccines
to diseases that haven&#39;t been possible to date,
things where the viruses rapidly evolve,
such with rhinovirus.
Wouldn&#39;t it be nice to have something that actually blocked common colds?
Or, more importantly, HIV,
where the virus evolves so quickly
the vaccines that are made today
can&#39;t keep up
with those evolutionary changes.
Also, at Synthetic Genomics,
we&#39;ve been working
on major environmental issues.
I think this latest oil spill in the Gulf
is a reminder.
We can&#39;t see CO2 --
we depend on scientific measurements for it
and we see the beginning results
of having too much of it --
but we can see pre-CO2 now
floating on the waters
and contaminating the beaches in the Gulf.
We need some alternatives
for oil.
We have a program with Exxon Mobile
to try and develop new strains of algae
that can efficiently capture carbon dioxide
from the atmosphere or from concentrated sources,
make new hydrocarbons that can go into their refineries
to make normal gasoline
and diesel fuel out of CO2.
Those are just a couple of the approaches
and directions that we&#39;re taking.

(Applause)

