
You know, I&#39;ve talked about some of these projects before --
about the human genome and what that might mean,
and discovering new sets of genes.

We&#39;re actually starting at a new point:
we&#39;ve been digitizing biology,
and now we&#39;re trying to go from that digital code
into a new phase of biology
with designing and synthesizing life.
So, we&#39;ve always been trying to ask big questions.
&quot;What is life?&quot; is something that I think many biologists
have been trying to understand
at various levels.
We&#39;ve tried various approaches,
paring it down to minimal components.
We&#39;ve been digitizing it now for almost 20 years;
when we sequenced the human genome,
it was going from the analog world of biology
into the digital world of the computer.
Now we&#39;re trying to ask, &quot;Can we regenerate life
or can we create new life
out of this digital universe?&quot;
This is the map of a small organism,
Mycoplasma genitalium,
that has the smallest genome for a species
that can self-replicate in the laboratory,
and we&#39;ve been trying to just see if
we can come up with an even smaller genome.
We&#39;re able to knock out on the order of 100 genes
out of the 500 or so that are here.
When we look at its metabolic map,
it&#39;s relatively simple
compared to ours --
trust me, this is simple --
but when we look at all the genes
that we can knock out one at a time,
it&#39;s very unlikely that this would yield
a living cell.
So we decided the only way forward
was to actually synthesize this chromosome
so we could vary the components
to ask some of these most fundamental questions.

And so we started down the road of:
can we synthesize a chromosome?
Can chemistry permit making
these really large molecules
where we&#39;ve never been before?
And if we do, can we boot up a chromosome?
A chromosome, by the way, is just a piece of inert chemical material.
So, our pace of digitizing life has been increasing
at an exponential pace.
Our ability to write the genetic code
has been moving pretty slowly
but has been increasing,
and our latest point would put it on, now, an exponential curve.
We started this over 15 years ago.
It took several stages, in fact,
starting with a bioethical review before we did the first experiments.
But it turns out synthesizing DNA
is very difficult.
There are tens of thousands of machines around the world
that make small pieces of DNA --
30 to 50 letters in length --
and it&#39;s a degenerate process, so the longer you make the piece,
the more errors there are.
So we had to create a new method
for putting these little pieces together and correct all the errors.
And this was our first attempt, starting with the digital information
of the genome of phi X174.
It&#39;s a small virus that kills bacteria.
We designed the pieces, went through our error correction
and had a DNA molecule
of about 5,000 letters.
The exciting phase came when we took this piece of inert chemical
and put it in the bacteria,
and the bacteria started to read this genetic code,
made the viral particles.
The viral particles then were released from the cells
and came back and killed the E. coli.
I was talking to the oil industry recently
and I said they clearly understood that model.

(Laughter)

They laughed more than you guys are. 
(Laughter)

And so, we think this is a situation
where the software can actually build its own hardware
in a biological system.

But we wanted to go much larger:
we wanted to build the entire bacterial chromosome --
it&#39;s over 580,000 letters of genetic code --
so we thought we&#39;d build them in cassettes the size of the viruses
so we could actually vary the cassettes
to understand
what the actual components of a living cell are.
Design is critical,
and if you&#39;re starting with digital information in the computer,
that digital information has to be really accurate.
When we first sequenced this genome in 1995,
the standard of accuracy was one error per 10,000 base pairs.
We actually found, on resequencing it,
30 errors; had we used that original sequence,
it never would have been able to be booted up.
Part of the design is designing pieces
that are 50 letters long
that have to overlap with all the other 50-letter pieces
to build smaller subunits
we have to design so they can go together.
We design unique elements into this.
You may have read that we put watermarks in.

Think of this:
we have a four-letter genetic code -- A, C, G and T.
Triplets of those letters
code for roughly 20 amino acids,
such that there&#39;s a single letter designation
for each of the amino acids.
So we can use the genetic code to write out words,
sentences, thoughts.
Initially, all we did was autograph it.
Some people were disappointed there was not poetry.
We designed these pieces so
we can just chew back with enzymes;
there are enzymes that repair them and put them together.
And we started making pieces,
starting with pieces that were 5,000 to 7,000 letters,
put those together to make 24,000-letter pieces,
then put sets of those going up to 72,000.
At each stage, we grew up these pieces in abundance
so we could sequence them
because we&#39;re trying to create a process that&#39;s extremely robust
that you can see in a minute.
We&#39;re trying to get to the point of automation.
So, this looks like a basketball playoff.
When we get into these really large pieces
over 100,000 base pairs,
they won&#39;t any longer grow readily in E. coli --
it exhausts all the modern tools of molecular biology --
and so we turned to other mechanisms.
We knew there&#39;s a mechanism called homologous recombination
that biology uses to repair DNA
that can put pieces together.

Here&#39;s an example of it:
there&#39;s an organism called
Deinococcus radiodurans
that can take three millions rads of radiation.
You can see in the top panel, its chromosome just gets blown apart.
Twelve to 24 hours later, it put it
back together exactly as it was before.
We have thousands of organisms that can do this.
These organisms can be totally desiccated;
they can live in a vacuum.
I am absolutely certain that life can exist in outer space,
move around, find a new aqueous environment.
In fact, NASA has shown a lot of this is out there.
Here&#39;s an actual micrograph of the molecule we built
using these processes, actually just using yeast mechanisms
with the right design of the pieces we put them in;
yeast puts them together automatically.
This is not an electron micrograph;
this is just a regular photomicrograph.
It&#39;s such a large molecule
we can see it with a light microscope.
These are pictures over about a six-second period.
So, this is the publication we had just a short while ago.
This is over 580,000 letters of genetic code;
it&#39;s the largest molecule ever made by humans of a defined structure.
It&#39;s over 300 million molecular weight.
If we printed it out at a 10 font with no spacing,
it takes 142 pages
just to print this genetic code.
Well, how do we boot up a chromosome? How do we activate this?
Obviously, with a virus it&#39;s pretty simple;
it&#39;s much more complicated dealing with bacteria.
It&#39;s also simpler when you go

into eukaryotes like ourselves:
you can just pop out the nucleus
and pop in another one,
and that&#39;s what you&#39;ve all heard about with cloning.
With bacteria and Archaea, the chromosome is integrated into the cell,
but we recently showed that we can do a complete transplant
of a chromosome from one cell to another
and activate it.
We purified a chromosome from one microbial species --
roughly, these two are as distant as human and mice --
we added a few extra genes
so we could select for this chromosome,
we digested it with enzymes
to kill all the proteins,
and it was pretty stunning when we put this in the cell --
and you&#39;ll appreciate
our very sophisticated graphics here.
The new chromosome went into the cell.
In fact, we thought this might be as far as it went,
but we tried to design the process a little bit further.
This is a major mechanism of evolution right here.
We find all kinds of species
that have taken up a second chromosome
or a third one from somewhere,
adding thousands of new traits
in a second to that species.
So, people who think of evolution
as just one gene changing at a time
have missed much of biology.
There are enzymes called restriction enzymes
that actually digest DNA.
The chromosome that was in the cell
doesn&#39;t have one;
the chromosome we put in does.
It got expressed and it recognized
the other chromosome as foreign material,
chewed it up, and so we ended up
just with a cell with the new chromosome.
It turned blue because of the genes we put in it.
And with a very short period of time,
all the characteristics of one species were lost
and it converted totally into the new species
based on the new software that we put in the cell.
All the proteins changed,
the membranes changed;
when we read the genetic code, it&#39;s exactly what we had transferred in.
So, this may sound like genomic alchemy,
but we can, by moving the software of DNA around,
change things quite dramatically.
Now I&#39;ve argued, this is not genesis;
this is building on three and a half billion years of evolution.
And I&#39;ve argued that we&#39;re about to perhaps
create a new version of the Cambrian explosion,
where there&#39;s massive new speciation
based on this digital design.
Why do this?
I think this is pretty obvious in terms of some of the needs.
We&#39;re about to go from six and a half
to nine billion people over the next 40 years.

To put it in context for myself:
I was born in 1946.
There are now three people on the planet
for every one of us that existed in 1946;
within 40 years, there&#39;ll be four.
We have trouble feeding, providing fresh, clean water,
medicines, fuel
for the six and a half billion.
It&#39;s going to be a stretch to do it for nine.
We use over five billion tons of coal,
30 billion-plus barrels of oil --
that&#39;s a hundred million barrels a day.
When we try to think of biological processes
or any process to replace that,
it&#39;s going to be a huge challenge.
Then of course, there&#39;s all that
CO2 from this material
that ends up in the atmosphere.
We now, from our discovery around the world,
have a database with about 20 million genes,
and I like to think of these as the design components of the future.
The electronics industry only had a dozen or so components,
and look at the diversity that came out of that.
We&#39;re limited here primarily
by a biological reality
and our imagination.
We now have techniques,
because of these rapid methods of synthesis,
to do what we&#39;re calling combinatorial genomics.
We have the ability now to build a large robot
that can make a million chromosomes a day.
When you think of processing these 20 million different genes
or trying to optimize processes
to produce octane or to produce pharmaceuticals,
new vaccines,
we can just with a small team,
do more molecular biology
than the last 20 years of all science.

And it&#39;s just standard selection:
we can select for viability,
chemical or fuel production,
vaccine production, etc.
This is a screen snapshot
of some true design software
that we&#39;re working on to actually be able to sit down
and design species in the computer.

You know, we don&#39;t know necessarily what it&#39;ll look like:
we know exactly what their genetic code looks like.
We&#39;re focusing on now fourth-generation fuels.
You&#39;ve seen recently, corn to ethanol
is just a bad experiment.
We have second- and third-generation fuels
that will be coming out relatively soon
that are sugar, to much higher-value fuels
like octane or different types of butanol.
But the only way we think that biology
can have a major impact without
further increasing the cost of food and limiting its availability
is if we start with CO2 as its feedstock,
and so we&#39;re working with designing cells to go down this road.
And we think we&#39;ll have the first fourth-generation fuels
in about 18 months.
Sunlight and CO2 is one method ...

(Applause)

but in our discovery around the world,
we have all kinds of other methods.
This is an organism we described in 1996.
It lives in the deep ocean,
about a mile and a half deep,
almost at boiling-water temperatures.
It takes CO2 to methane
using molecular hydrogen as its energy source.
We&#39;re looking to see if we can take
captured CO2,
which can easily be piped to sites,
convert that CO2 back into fuel
to drive this process.
So, in a short period of time,
we think that we might be able to increase
what the basic question is of &quot;What is life?&quot;
We truly, you know,
have modest goals
of replacing the whole petrol-chemical industry --

(Laughter)
 
(Applause)

Yeah. If you can&#39;t do that at TED, where can you? --

(Laughter)

become a major source of energy ...
But also, we&#39;re now working on using these same tools
to come up with instant sets of vaccines.
You&#39;ve seen this year with flu;
we&#39;re always a year behind and a dollar short
when it comes to the right vaccine.
I think that can be changed
by building combinatorial vaccines in advance.
Here&#39;s what the future may begin to look like
with changing, now, the evolutionary tree,
speeding up evolution
with synthetic bacteria, Archaea
and, eventually, eukaryotes.

We&#39;re a ways away from improving people:
our goal is just to make sure that we have a chance
to survive long enough to maybe do that. Thank you very much.

(Applause)

